app/labor/cache_buster.rb

```diff
@@ -5,11 +5,8 @@ class CacheBuster
 
   def bust(path)
     return unless Rails.env.production?
-    request = HTTParty.post("https://api.fastly.com/purge/https://dev.to#{path}",
-    headers: { "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"] })
-    request = HTTParty.post("https://api.fastly.com/purge/https://dev.to#{path}?i=i",
-    headers: { "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"] })
-    request
+    HTTParty.post("https://api.fastly.com/purge/https://dev.to#{path}?i=i",
```

Comment:
This is changing behavior though.

The method on the left does two requests and you removed one... You can remove the assignments but should keep the two calls...

Reply:
oops...
thanks review!

Reply:
Good call.

@koheisg was right to smell this as weird, as a lot of the stuff we're doing with cachebusting is hacky based on my inability to get the normal behavior of the [fastly-rails](https://github.com/fastly/fastly-rails) gem to work about two years ago. I'll write some clarifying guidance on this soon.